<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IntonationParametersWindow</class>
 <widget class="QWidget" name="IntonationParametersWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Intonation parameters</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Parameters</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QCheckBox" name="useParametersCheckBox">
        <property name="text">
         <string>Use these parameters</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Notional pitch:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="notionalPitchSpinBox"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Pretonic pitch range:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDoubleSpinBox" name="pretonicPitchRangeSpinBox"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Pretonic perturbation range:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QDoubleSpinBox" name="pretonicPerturbationRangeSpinBox"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Tonic pitch range:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QDoubleSpinBox" name="tonicPitchRangeSpinBox"/>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Tonic perturbation range:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QDoubleSpinBox" name="tonicPerturbationRangeSpinBox"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="updateButton">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Update</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0" rowspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Control</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="microCheckBox">
        <property name="text">
         <string>Micro intonation</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="macroCheckBox">
        <property name="text">
         <string>Macro intonation</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="smoothCheckBox">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Smooth intonation</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="randomCheckBox">
        <property name="text">
         <string>Random intonation</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="driftCheckBox">
        <property name="text">
         <string>Intonation drift</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Drift parameters</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="2">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Hz</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>semitone</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="deviationSpinBox"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Lowpass cutoff:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="cutoffSpinBox"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Deviation:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>microCheckBox</tabstop>
  <tabstop>macroCheckBox</tabstop>
  <tabstop>smoothCheckBox</tabstop>
  <tabstop>randomCheckBox</tabstop>
  <tabstop>driftCheckBox</tabstop>
  <tabstop>useParametersCheckBox</tabstop>
  <tabstop>notionalPitchSpinBox</tabstop>
  <tabstop>pretonicPitchRangeSpinBox</tabstop>
  <tabstop>pretonicPerturbationRangeSpinBox</tabstop>
  <tabstop>tonicPitchRangeSpinBox</tabstop>
  <tabstop>tonicPerturbationRangeSpinBox</tabstop>
  <tabstop>deviationSpinBox</tabstop>
  <tabstop>cutoffSpinBox</tabstop>
  <tabstop>updateButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
